templates/example_MPH_plots.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    total_counts = bin_files.get_total_counts(bin_file_path)
</pre><p>This should also be modified to work one FOV at a time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>